tests/api/test_api.py

```diff
@@ -0,0 +1,19 @@
+import pytest
+import requests
+
+class TestAPI:
+    url_pokemon="https://pokeapi.co/api/v2/pokemon/bulbasaur/"
+    url_type="https://pokeapi.co/api/v2/type/poison/"
```

Comment:
Small comments here:

- Since the Domain is the same, why don't we trim the domain?